SQL/SQL.sublime-syntax

```diff
@@ -222,3 +223,14 @@ contexts:
             0: punctuation.definition.string.end.sql
           pop: true
         - include: string_interpolation
+  identifier_create:
+    - meta_content_scope: entity.name.function.sql
+    - match: \w+
+    - match: \'[^']\'
+    - match: \"[^"]\"
+    - match: \`[^`]\`
```

Comment:
The source had ``''(\w+)''|"(\w+)"|`(\w+)` `` for these which would require a `+` after the sets.

A test case for at least one of these seems appropriate.

Reply:
well spotted, thanks. I have updated the match patterns and included some extra test cases.